doc/source/whatsnew/v0.21.0.txt

```diff
@@ -328,6 +327,10 @@ Deprecations
 
 - ``pd.options.html.border`` has been deprecated in favor of ``pd.options.display.html.border`` (:issue:`15793`).
 
+
+- :func: `to_datetime` now takes ``tz`` keyword argument, ``utc`` argument is deprecated (:issue:`13712`)
```

Comment:
Say something like (don't copy, make sure to format): The "utc" argument in "to_datetime" has deprecated in favor of "tz"